src/base/static/js/views/app-view.js

```diff
@@ -163,6 +165,7 @@ export default Backbone.View.extend({
     );
 
     const storeState = store.getState();
+    // TODO: refactor and clean up this logic into a separate theme provider
```

Comment:
I think I'll create a separate PR for this later, which should simplify the multiple theme HOC's from Material-UI and Emotion.

Reply:
New PR that cleans this up is here: https://github.com/mapseed/platform/pull/1179